sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeIdentityDocumentsFromUrl.java

```diff
@@ -17,10 +17,9 @@
 
 /**
  * Sample for recognizing commonly found ID document fields from a file source URL of an invoice document.
```

Comment:
we're recognizing on a license in this sample but it says "invoice document"

Reply:
nice catch. It turned out it might be we did copy and pasted directly in preview PRs.